angular-site/src/app/buttons-and-indicators/progress-bar/progress-bar-buffer-example.html

```diff
@@ -3,7 +3,7 @@ <h2 class="example-h2">
     Buffer progress-bar
   </a>
 </h2>
-<button mat-raised-button (click)="showProgressBar(5)">
+<button mat-raised-button (click)="showProgressBar(5)" id="progress_bars-show_buffer_button">
```

Comment:
Please avoid underscore in ids.
Hint. For example, if you named your progress-bar variable something like
**public static Progressbar bufferProgressbar;**
and id of the element as
**id="buffer-progressbar"**
then it is no need mark element with annotation
_@UI("#buffer-progressbar")_ or _@Css("#buffer-progressbar")_